tests/test_minimal/test_tools/test_expand_paths.py

```diff
@@ -98,3 +99,62 @@ def test_expand_paths(tmpdir):
     for x in string_directory_out:
         assert x in expected
     assert len(string_directory_out) == len(expected)
+
+
+def test_expand_paths_ibl(tmpdir):
+    expander = LocalPathExpander()
+
+    # set up IBL Steinmetz video file structure
+    generate_path_expander_demo_ibl(tmpdir)
+    base_directory = Path(tmpdir)
+
+    # NOTE: empty brackets used b/c for some reason NR_0021/2022-06-30/ subdir is 002/ instead of 001/
+    out = expander.expand_paths(
+        dict(
+            ibl_video_file=dict(
+                base_directory=base_directory,
+                file_path="steinmetzlab/Subjects/{subject_id}/{session_start_time:%Y-%m-%d}/{}/raw_video_data/_iblrig_leftCamera.raw.{session_id}.mp4",
+            ),
+            ibl_video_directory=dict(
+                base_directory=base_directory,
+                folder_path="steinmetzlab/Subjects/{subject_id}/{session_start_time:%Y-%m-%d}/{}/raw_video_data",
+            ),
+        ),
+    )
+
+    # build expected output with glob and manual parsing
+    expected = []
+    for file_path in base_directory.glob("steinmetzlab/Subjects/*/*/*/raw_video_data/_iblrig_leftCamera.raw.*.mp4"):
+        subject_id = file_path.parts[-5]
+        session_start_time = datetime.strptime(file_path.parts[-4], "%Y-%m-%d")
+        session_id = file_path.parts[-1].split(".")[-2]
+        expected.append(
```

Comment:
We try to make our expected out come cases in tests as explicit as possible; what I would recommend is to take the returned dictionary from the `PathExpander` output and save it as a `.json`, then have this test load the dictionary back for testing

That way we can look at the full output structure outside of code to make sure it looks as truly expected (not that I have any doubts, just something we're in the habit of doing these days)